### DISCUSSION TITLE
can`t running

### DISCUSSION THREAD

**Discussion by EpsilonYi** [vuetifyjs/vuetify#19056] [General] [answered]
![屏幕截图 2024-01-16 210020](https://github.com/vuetifyjs/vuetify/assets/16634792/7b45f1c1-e0ca-4840-98f1-7e6887bbb36e)

**Comment by johnleider** [comment, marked as answer]
Are you using Windows? If so, change your dev script to "vite" only. You will get a warning message because we import json and it's technically an experimental feature. If you don't want to see the error, you can install "cross-env" and then update your dev script to "cross-env node --no-warnings ./node_modules/.bin/vite"
